group_members_test.go

```diff
@@ -48,3 +49,49 @@ func TestListBillableGroupMembers(t *testing.T) {
 		t.Errorf("Groups.ListBillableGroupMembers returned %+v, want %+v", billableMembers, want)
 	}
 }
+
+func TestListGroupMembersWithoutSAML(t *testing.T) {
+	mux, server, client := setup(t)
+	defer teardown(server)
+
+	mux.HandleFunc("/api/v4/groups/1/members",
+		func(w http.ResponseWriter, r *http.Request) {
+			testMethod(t, r, http.MethodGet)
+			fmt.Fprint(w, `[{"id": 1,"username": "raymond_smith","name": "Raymond Smith","state": "active","avatar_url": "https://www.gravatar.com/avatar/c2525a7f58ae3776070e44c106c48e15?s=80&d=identicon","web_url": "http://192.168.1.8:3000/root","created_at": "2012-10-21T14:13:35Z","expires_at": "2012-10-22T14:13:35Z","access_level": 30,"group_saml_identity": null}]`)
+		})
```

Comment:
Can you format these so it's a bit more readable?

Reply:
Updated with commit.